src/auto-evo/simulation/PopulationSimulation.cs

```diff
@@ -130,23 +133,115 @@ private static void SimulatePatchStep(RunResults populations, Patch patch, List<
             bool lowSpecies = species.Count <= Constants.AUTO_EVO_LOW_SPECIES_THRESHOLD;
             bool highSpecies = species.Count >= Constants.AUTO_EVO_HIGH_SPECIES_THRESHOLD;
```

Comment:
Shouldn't this be removed from here? Now that there is kinda an auto evo algorithm this hack to keep the species count reasonable shouldn't be needed.